js/calendar.js

```diff
@@ -1,43 +1,39 @@
-export function renderCalendarCells() {
-    document.addEventListener('DOMContentLoaded', () => {
-        const calendarCells = document.getElementById('calendarCells');
-        const cellsInOneColumn= 19;
-        const daysInWeek = 7;
-        for (let i = 0; i < cellsInOneColumn * daysInWeek; i++) {
-            const cell = document.createElement('div');
-            cell.className = 'cell';
-            calendarCells.appendChild(cell);
-        }
-    });
-}
-function getStartOfWeek(date) {
-    const day = date.getDay(); 
-    const diff = date.getDate() - day + (day === 0 ? -6 : 1); 
-    return new Date(date.setDate(diff));
-}
-function isToday(date) {
-    const today = new Date();
-    return date.getDate() === today.getDate() &&
-           date.getMonth() === today.getMonth() &&
-           date.getFullYear() === today.getFullYear();
+import { getStartOfWeek, isToday } from "./utils.js";
+
+export let startDayOfWeek = new Date();
```

Comment:
I don't see it used in other file so no export needed.
also, since value is current date, naming is not correct. if it is current date I do not think you should change it, so it should be const.

In case you need variable where you save same value that you change in loop, it should be undefined at first, before you assign value. 
then if I understand correctly what you use this variable for it could be named like weekDay or smth like that : )

```suggestion
const currentDate = new Date();
```